dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassFileTest.java

```diff
@@ -21,27 +21,28 @@
 import com.google.common.testing.EqualsTester;
 import com.google.common.testing.NullPointerTester;
 import com.google.common.testing.NullPointerTester.Visibility;
+import java.io.IOException;
 import java.nio.file.Paths;
 import org.junit.Test;
 
 public class ClassFileTest {
   @Test
-  public void testCreation() {
+  public void testCreation() throws IOException {
     ClassPathEntry entry = new ClassPathEntry(Paths.get("foo", "bar.jar"));
     ClassFile file = new ClassFile(entry, "com.test.Foo");
     assertEquals("com.test.Foo", file.getBinaryName());
     assertEquals(new ClassPathEntry(Paths.get("foo", "bar.jar")), file.getClassPathEntry());
   }
 
   @Test
-  public void testNull() {
+  public void testNull() throws IOException {
     new NullPointerTester()
         .setDefault(ClassPathEntry.class, new ClassPathEntry(Paths.get("foo", "bar.jar")))
         .testConstructors(ClassFile.class, Visibility.PACKAGE);
   }
 
   @Test
-  public void testEquality() {
+  public void testEquality() throws IOException {
```

Comment:
It looks strange to throw IOException just to checking equality.

Reply:
fixed